packages/react-native/src/primitives/PhoneNumberField/styles.ts

```diff
@@ -0,0 +1,28 @@
+import { Platform, StyleSheet } from 'react-native';
+
+import { PhoneNumberFieldStyles } from './types';
+
+export const styles: PhoneNumberFieldStyles = StyleSheet.create({
+  container: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+  },
+  fieldStyle: {
+    flex: 1,
+  },
+  picker: {
+    flex: 1,
+    /*
+       arbitrary number based off of the legacy amplify react-native component
+       this value MUST match the height of the pickerItem
+    */
+    height: 44,
```

Comment:
Since this value must match the height of the `pickerItem`, can we put it in a constant?